recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/domd-salvator-x-m3n.cfg

```diff
@@ -0,0 +1,382 @@
+# =====================================================================
```

Comment:
if this one is dummy just put a file with comment that the actual content will be added later, no need to bring lots of stuff beforehand?

Reply:
Done